config/src/main/java/se/fortnox/reactivewizard/config/ConfigReader.java

```diff
@@ -56,6 +56,9 @@ private static String replaceEnvPlaceholders(String config) {
         StringBuffer        stringBuffer = new StringBuffer();
         do {
             String value = env.get(matcher.group(1));
+            if (value != null) {
+                value = value.replace("\\n", "\n");
```

Comment:
Wouldn't this cause problems when you have strings that are supposed to contain an escaped newline?
like `emailBody: "Hi\\nWelcome to thing!"`

Reply:
Tried searching for escaped newlines but could not find any results https://git.fnox.se/plugins/servlet/search?q=%22%5C%5Cn%22

Reply:
Do we really need to support escaped newlines inside YAML's?

Reply:
Do we have any example of newlines being the expected outcome of reading the yaml? Since this will be breaking those configs. Otherwise if that becomes a problem we could add an escape sequence that's used for actual newlines.

Reply:
After some discussions we've come to believe that there are no good reason for having escaped newlines like that, so we're gonna go ahead and try this out.
If this doesn't work there is a workaround in the featureswitch service that can be used.